.github/workflows/documentation.yaml

```
@@ -97,13 +100,28 @@ jobs:
         shell: Rscript {0}
 
 
-        ##### Build documentation and put it on "gh-pages" branch
+        ##### Build documentation and put it on "rpolars/rpolars.github.io" repo
 
       - name: Build docs folder
         run: |
           altdoc::update_docs(custom_reference = "docs/make-docs.R")
         shell: Rscript {0}
 
+      - name: Build site
+        run: cd docs && mkdocs build
+
+      - uses: webfactory/ssh-agent@v0.6.0
+        if: github.event_name != 'pull_request'
+        with:
+          ssh-private-key: ${{ secrets.DEPLOY_DOCS }}
+
         # https://www.mkdocs.org/user-guide/deploying-your-docs/
       - name: Build site and deploy to GitHub pages
-        run: cd docs && mkdocs gh-deploy --force
+        if: github.event_name != 'pull_request'
+        uses: JamesIves/github-pages-deploy-action@v4
+        with:
+          clean: false
```

Perhaps if this is not set to `true`, the pkgdown files will continue to remain.

```suggestion
          clean: true
```

In this case, we also need to make sure that files like `.nojekyll` continue to remain.